cypress/integration/upload.spec.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * SPDX-FileCopyrightText: 2020 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+const url = 'http://localhost:3001/test-image.png'
```

Comment:
Can't we get this value from cypress?

Reply:
no, because it does not exist. It could literally be anything like 'foobar'.

Reply:
I mean the base URL

Reply:
I changed it to make it more clear that this is mock value